plugins/modules/deb_publication.py

```diff
@@ -26,6 +26,22 @@
       - Version number to be published
     type: int
     required: false
+  simple:
+    description:
+      - Whether to use simple mode. If none of simple, structured or verbatim
+        are specified, then a simple publication will be created
```

Comment:
I would leave a note that these are incompatible with verbatim (if we keep the options that way).